### ISSUE TITLE
add support for 96x96 cursors

### ISSUE THREAD

**Issue by jsorg71** [neutrinolabs/xrdp#2259]
xrdp currently only support 32x32 cursors, current protocol supports 96x96

**Comment by matt335672** [comment]
Possible other related issues are : #716 #1333.

**Comment by rbreaves** [comment]
Can also be a good work around for users that experience double cursors.. or vice versa if either one of these issues get fixed.
https://github.com/neutrinolabs/xrdp/issues/2046

**Comment by pwn0r** [comment]
yes please. the simplest example how to exceed cursor size on xrdp -- on ubuntu 22.04 (or whatever recent version) enable a cursor in accessibility menu.  *medium* is the largest size that can be used without clipping on xrdp.

**Comment by matt335672** [comment]
See #2315 which will address this issue for the Xorg backend
